tools/downloader/common.py

```diff
@@ -480,7 +481,7 @@ def apply(self, reporter, output_dir):
 
 class Model:
     def __init__(self, name, subdirectory, files, postprocessing, mo_args, quantizable, framework,
-                 description, license_url, precisions, task_type, conversion_to_onnx_args):
+                 description, license_url, precisions, task_type, conversion_to_onnx_args, compilable = False):
```

Comment:
```suggestion
                 description, license_url, precisions, task_type, conversion_to_onnx_args):
```

Reply:
Ok

Reply:
Why you removed `compilable` arg from init?

Reply:
My bad, I just meant to remove the default value.